### ISSUE TITLE
Inconsistent `.native` conversion for broken certs

### ISSUE THREAD

**Issue by rooterkyberian** [wbond/asn1crypto#103]
If I try to convert certificate to `native` while it contains invalid fields I get and error first time around, and only first time:

Broken cert example taken from:  https://github.com/pyca/cryptography/blob/a5865d9b630df7586785b683ed145b977ad7e0c4/vectors/cryptography_vectors/x509/san_x400address.der
```python
from asn1crypto import pem, x509

with open('san_x400address.der', 'rb') as f:
    der_bytes = f.read()

cert = x509.Certificate.load(der_bytes)

for i in range(2):
    print(i)
    try:
        result = cert.native
    except Exception as e:
        result = e
    print(result)
```
**I get:**
```
0
Error parsing asn1crypto.x509.BuiltInStandardAttributes - method should have been constructed, but primitive was found
    while parsing asn1crypto.x509.ORAddress
    while parsing asn1crypto.x509.GeneralNames
    while parsing asn1crypto.x509.Extension
    while parsing asn1crypto.x509.Extensions
    while parsing asn1crypto.x509.TbsCertificate
    while parsing asn1crypto.x509.Certificate
1
OrderedDict()
```

**Expected:** both times I should get the same error.

Tested with `asn1crypto==0.24.0`.

**Comment by rooterkyberian** [comment]
For now I'm doing a `copy.deepcopy` on a object before I try to convert it to native to counteract this, but obviously it is suboptimal.

**Comment by wbond** [comment]
Thanks for the bug report!

**Comment by wbond** [comment]
This should be fixed by c68ed760d00e25564e62fc1335f9794d37e39e50
